index.html

```diff
@@ -188,6 +191,28 @@ <h5>Skills</h5>
                         </div>
                     </div>
                 </section>
+                <footer>
+                    <section id="contact-form">
+                        <div class="contact-details">
+                            <h2>Contact me</h2>
+                            <p>If you have an application you are interested in developing, a feature that you need
+                                built or a project that needs coding. I’d love to help with it!</p>
+                        </div>
+                        <div class="form">
+                            <form action="https://formspree.io/f/xzbqjwon" method="post">
+                                <input type="text" name="name" placeholder="Lorem Ipsum" maxlength="30" required>
+                                <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                                <div class="container">
+                                    <textarea type="text" placeholder="Write your message here..." rows="10"
+                                        columns="50" maxlength="500"></textarea>
+                                </div>
+                                <div class="someter">
+                                    <button>Get in touch</button>
+                                </div>
+                            </form>
+                        </div>
+                    </section>
+                </footer>
```

Comment:
- Your design is awesome :clap: :clap: but there are  a  few things that could be improved to make it match the design more closely, and to make it a great design to pride about with friends :smile: 
_Check this:_

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/235742056-287f536c-d7af-49bc-9414-c92fa3ecddeb.png)|![image](https://user-images.githubusercontent.com/99715640/235742128-5f5380b0-6446-49e2-b02c-ba3d8ec50aa0.png)|
  - Looking at the figure above, one can almost notice a sharp contrast of:
    - [ ] The missing `black divider` at the bottom base of the page, and
    - [ ] The overflow indicated on the `x-axis` of your page when on a mobile screen of `375px` due to an element with some fixed width that's spanning beyond normal screen size. This is not really good for user experience as they will have to struggle with scrolling sideways before enjoying the beauty of interacting with your page :scream: 

_**I am confident that this won't take time to complete, with your expertise. Let's get this done :muscle:**_

Reply:
This Black Line is Just Line or the scrollbar?

Reply:
It's just an aesthetic line signifying the end of the page. Having a scroll ball on a mobile device won't be good experience for users.

Reply:
so Sir you mean i should remove overflow-x? idk why on my pc it wont show overflow?

Reply:
Can you inspect the page, and adjust the width to 375px? Or if you prefer the mobile rendering, just drag the mouse towards the left hand side as if you are dragging on a screen touch, that excess space outside the standard width is what I think you need to remove.

Did you see where I placed a red arrow pointing rightward on the image that I uploaded? That's what I'm talking about.